rtloader/demo/main.c

```diff
@@ -26,7 +28,7 @@ char **get_tags(char *id, int highCard)
     return data;
 }
 
-void submitMetric(char *id, metric_type_t mt, char *name, double val, char **tags,  char *hostname)
+void submitMetric(char *id, metric_type_t mt, char *name, double val, char **tags,  char *hostname, bool flush_first_val)
```

Comment:
`flush_first_val` is unused.

Reply:
true but it is required in the callback otherwise it doesn't compile

Reply:
I assumed the code compile before your changes and you just added a new parameter!